KitePrintSDK/src/main/java/ly/kite/print/BaseRequest.java

```diff
@@ -106,7 +119,7 @@ protected JSONHttpResponse doInBackground(Void... voids) {
                     }
                 }
 
-                request.setHeader("Authorization", "ApiKey " + KitePrintSDK.getAPIKey() + ":");
+                request.setHeader("Authorization", "ApiKey " + KiteSDK.getInstance( mContext ).getAPIKey() + ":");
 
```

Comment:
Ah I've just remembered we have a few additional headers on iOS that get sent along with all Kite requests that should be replicated on Android, namely `User-Agent`, `X-App-Bundle-Id` (which should probably be `X-App-Package-Name` on Android?) & `X-App-Name`:

```[request setValue:[NSString stringWithFormat: @"Kite SDK iOS v%@", kOLKiteSDKVersion] forHTTPHeaderField:@"User-Agent"];
    [request setValue:[[NSBundle mainBundle] bundleIdentifier] forHTTPHeaderField:@"X-App-Bundle-Id"];
    [request setValue:[self appName] forHTTPHeaderField:@"X-App-Name"];```